http://deb.debian.org/debian/pool/main/libg/libgenome/libgenome_1.3.11+svn20110227.4616-3.debian.tar.xz
Author: Andreas Tille <tille@debian.org>
Last-Update: Fri, 12 Aug 2016 10:35:40 +0200
Bug-Debian: https://bugs.debian.org/831188
Description: THis declaration makes gcc-6.1 fail so simply drop it

--- a/libGenome/gnDefs.cpp
+++ b/libGenome/gnDefs.cpp
@@ -4,8 +4,4 @@
 
 #include "libGenome/gnDefs.h"
 
-// some compilers don't have abs() for 64 bit ints
-int64 abs( int64 a ){
-	return a < 0 ? -a : a;
-}
 
